src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './styles.css';
+
+const Footer = (props) => {
+  return (
+    <div className="Footer-container">
+      <div className="Footer">
+        <div className="Icon">
+          <i className="fas fa-home"/>
+        </div>
+        <div className="Icon">
+          <i className="fas fa-search"/>
+        </div>
+        <div className="Icon">
+          <i className="far fa-plus-square"/>
+        </div>
+        <div className="Icon">
+          <i className="far fa-heart"/>
+        </div>
+        <div className="Icon">
+          <i className="fas fa-user"/>
+        </div>
+      </div>
+    </div>
+  );
+}
+  
+  export default Footer;
+
```

Comment:
so many lines after `export default`. we should have only one new line at the end of file

Reply:
done

Reply:
done